transport-native-epoll/src/test/java/io/netty/channel/unix/SocketTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.unix;
+
+import io.netty.channel.epoll.Epoll;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class SocketTest {
+    static {
+        Epoll.ensureAvailability();
+    }
+
+    @Test
+    public void testKeepAlive() throws Exception {
+        Socket socket = Socket.newSocketStream();
```

Comment:
consider making this a member variable and using a `@Before` to initialize and `@After` to tear it down.